public/locales/en.json

```diff
@@ -4,6 +4,9 @@
         "title": "Collaborative markdown notes"
     },
     "renderer": {
+        "highlightCode": {
+            "copyCode": "Copy code"
```

Comment:
Despite the icon being easily identifiable, I suggest changing this to "Copy to clipboard".

Reply:
done